doc/interop-test-descriptions.md

```diff
@@ -1005,14 +1005,26 @@ Client asserts:
 ### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The client records the latency and status of each RPC in some data structure.
-If the test ever consumes `soak_overall_timeout_seconds` seconds and still hasn't
-completed `soak_iterations` RPCs, then the test should discontinue sending RPCs
-as soon as possible. After performing all RPCs, the test should examine
-previously recorded RPC latency and status results in a second pass and fail if
-either:
+The test can run in either a concurrent or non-concurrent mode, depending on 
+the number of threads specified (`soak_num_threads`):
 
-a) not all `soak_iterations` RPCs were completed
+* Non-Concurrent Mode: When `soak_num_threads = 1`, all RPCs are performed 
```

Comment:
not need to separately document concurrent mode and non-concurrent mode.

The behavior in both cases is overall the same. I.e. "non-concurrent" mode is now just the special case where num_threads=1